src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1587,13 +1629,49 @@ codet java_bytecode_convert_methodt::convert_instructions(
     }
     else if(statement=="iinc")
     {
+      code_blockt block;
+      block.add_source_location()=i_it->source_location;
+      // search variable on stack
+      const exprt &locvar=variable(arg0, 'i', i_it->address, CAST_AS_NEEDED);
+      for(size_t i=0; i<stack.size(); i++)
+      {
+        exprt &stack_entry=stack[i];
+        if(stack_entry.id()==ID_typecast)
+        {
+          const typecast_exprt &cast=to_typecast_expr(stack_entry);
+          stack_entry=cast.op();
```

Comment:
Make this code more similar to the above version; probably both should be: `stack_entry=to_typecast_expr(stack_entry).op();`